rust/fury-core/src/serializer/string.rs

```diff
@@ -37,8 +38,8 @@ impl Serializer for String {
         Ok(context.reader.string(len as usize))
     }
 
-    fn ty() -> FieldType {
-        FieldType::STRING
+    fn ty(_fury: &Fury) -> i16 {
```

Comment:
What does `ty` means?

Reply:
Yes. but type is a keyword, and type_id is a member of Any which is available for every type. ty is not a good  choise, Maybe we can use fury_type_id